.betterer.results

```diff
@@ -3899,9 +3899,7 @@ exports[`better eslint`] = {
     ],
     "public/app/features/explore/Logs/Logs.tsx:5381": [
       [0, 0, 0, "Unexpected any. Specify a different type.", "0"],
-      [0, 0, 0, "No untranslated strings. Wrap text with <Trans />", "1"],
-      [0, 0, 0, "No untranslated strings. Wrap text with <Trans />", "2"],
-      [0, 0, 0, "No untranslated strings. Wrap text with <Trans />", "3"]
+      [0, 0, 0, "No untranslated strings. Wrap text with <Trans />", "1"]
```

Comment:
I'm not sure which string in Logs.tsx hasn't been marked for translation. It would be helpful if betterer could point me to a specific string for which it thinks should be marked

Reply:
@harisrozajac betterer can't*, but eslint can. if you add `/* eslint @grafana/no-untranslated-strings: "error" */` to the top of the file it will show where the error is. in this case, looks like it's the unicode `!`. recommendation would be to use the `<Icon />` component

*_well... it can, but we've had to disable it for other reasons_